content/blog/2023-05-02-response-to-bitcoin-magazine-article-about-tokenization/index.md

```diff
@@ -0,0 +1,124 @@
+
+However, the demand for tokenizing assets has nothing to do with the distortions that the fiat monetary system has created. The tokenization of physical assets external to the blockchain on systems as Ethereum Classic is a change of paradigm of property registry systems.
+
+It is very likely that in the future, all sort of property records, from real estate, motor vehicles, corporate books, to shipping containers in the supply chain will be moved to the blockchain.
+
+### Mr. Koss Wrote:
+
+*“Blockworks highlighted this “advancement” in technology without properly addressing the potential side effects that widespread asset tokenization may cause. In the comments section, tokenization is touted as a means for individuals who are unable to purchase a home of their own to participate in the real estate market. But why are houses so expensive in the first place?”*
+
+### Our response:
+
+Widespread tokenization will happen regardless of side effects because blockchains are permissionless. And, tokenization will happen because they are an upgrade to property systems. In regions such as Latin America, Africa, and even in North America, the lack of security of municipal and state property records have been a source of underdevelopment, theft, and extremely high transactional costs.
+
+With its transparency, speed, and low transactional costs, the blockchain promises to significantly improve property registries, transfers, and thus asset markets worldwide.
+
+Real estate tokenization is not about investment or giving access to the “little guy” to participate. We agree with this assessment. Tokens that represent physical assets are about better title systems, with reduced corruption, theft, and confiscation.
+
+### Mr. Koss Wrote:
+
+*“[Houses are so expensive in the first place] because they are being used as stores of value, a former function of fiat money that is no longer possible due to decades of fiscal and monetary alchemy that has decimated peoples’ purchasing power.”*
+
+### Our response:
+
+We agree that houses are expensive in the first place, way higher than their utility value, because of decades of fiscal and monetary alchemy and outright fiat induced corruption. But, this is a problem totally unrelated to the blockchain or where the assets are registered and transferred. 
+
+A programmable blockchain as ETC, which is also proof of work and has a fixed monetary policy, is especially capable to take on the task of managing property records. One of the basic problems in impoverished nations are their unreliable registries.
+
+Programmable proof of work blockchains as ETC solve this by providing and incredibly secure and cheap environment to keep those records with a high degree of transparency and thus accountability. This will likely significantly reduce the problems that have haunted many cultures for centuries.
```

Comment:
Line 64: Add quantifier; Fix typo
```suggestion
Programmable proof of work blockchains such as ETC solve this by providing an incredibly secure and cheap environment to keep those records with a high degree of transparency and thus accountability. This will likely significantly reduce the problems that have haunted many cultures for centuries.
```